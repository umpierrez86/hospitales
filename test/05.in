leer_instancia
3 5
1 1 2
0 1 2 3 4
3 4 0 2 1
4 3 2 0 1
0 1 2
2 1 0
1 0 2
1 0 2
2 0 1
imprimir_instancia
asignar_residencias
imprimir_asignacion
Fin