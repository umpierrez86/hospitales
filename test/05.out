1>2>Cantidad hospitales: 3
Cupos hospitales: 1 1 2 
Cantidad estudiantes: 5
Preferencias hospitales
Preferencias hospital h0: 0 1 2 3 4 
Preferencias hospital h1: 3 4 0 2 1 
Preferencias hospital h2: 4 3 2 0 1 
Preferencias estudiantes
Preferencias estudiante e0: 0 1 2 
Preferencias estudiante e1: 2 1 0 
Preferencias estudiante e2: 1 0 2 
Preferencias estudiante e3: 1 0 2 
Preferencias estudiante e4: 2 0 1 
3>4>(0, 0) (1, libre) (2, 2) (3, 1) (4, 2) 
5>Fin.
